features/discovery/src/main/resources/META-INF/opennms/applicationContext-discovery.xml

```diff
@@ -46,6 +46,15 @@
     <property name="ipAddressFilter" ref="unmanagedInterfaceFilter"/>
   </bean>
 
+  <bean id="discoveryJobs" class="org.opennms.netmgt.discovery.messages.DiscoveryJob">
+	<!-- DiscoveryJob constructor is expecting List<IPPollRange> ranges, String foreignSource, String location, need to confirm from where I can get these values -->
```

Comment:
please let me know from where I can get the values which I need to pass in DiscoveryJob constructor